lib/shopify_api/resources/product.rb

```diff
@@ -16,6 +16,11 @@ def price_range
       end
     end
 
+    def total_inventory=(new_value)
+      raise(ShopifyAPI::ValidationException, 'deprecated behaviour') unless Base.api_version < ApiVersion.find_version('2019-10')
```

Comment:
Maybe the exception messages should point to https://help.shopify.com/en/api/guides/inventory-migration-guide?